src/rard/research/views/work.py

```diff
@@ -83,18 +88,48 @@ def get_success_url(self, *args, **kwargs):
 
     def form_valid(self, form):
         work = form.save(commit=False)
-        updated = form.cleaned_data["antiquarians"]
-        existing = work.antiquarian_set.all()
-        to_remove = [a for a in existing if a not in updated]
-        to_add = [a for a in updated if a not in existing]
-        for a in to_remove:
-            a.works.remove(work)
-        for a in to_add:
-            a.works.add(work)
-        add_new_books_to_work(work, form)
+        if "antiquarians" in form.cleaned_data:
+            updated = form.cleaned_data["antiquarians"]
+            existing = work.antiquarian_set.all()
+            to_remove = [a for a in existing if a not in updated]
+            to_add = [a for a in updated if a not in existing]
+            for a in to_remove:
+                a.works.remove(work)
+            for a in to_add:
+                a.works.add(work)
+            add_new_books_to_work(work, form)
         return super().form_valid(form)
 
 
+class WorkUpdateIntroductionView(WorkUpdateView):
```

Comment:
I'm seeing this when I click Add Introduction on the Work edit page:
<img width="482" alt="image" src="https://github.com/UCL/frrant/assets/5113832/c2b83ac2-c43e-45ae-b2c7-ee10623326a6">

Reply:
Same for the book

Reply:
Ok, ignore that, it was me. Privacy Badger had identified Quill.js as a potential tracker